engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/TestBase.java

```diff
@@ -146,6 +148,27 @@ protected static Value evalModule(Context ctx, String src) {
     return mainMethod.execute();
   }
 
+  /** Evaluates a project from tests resources. */
```

Comment:
Intrinsic method. I wonder how exactly one uses it? I guess I'll find out by reading rest of this PR...

Reply:
Sorry, this is a left-over method. It is no longer used anywhere, so I have removed it in 07a550457b919ff6cc5b52c4961cc7c0e6428764